vm/actor/src/builtin/miner/miner_state.rs

```diff
@@ -0,0 +1,993 @@
+// Copyright 2020 ChainSafe Systems
```

Comment:
Question: why was this file renamed from `state.rs`? It doesn't follow the convention used for other actors, which is just that the `miner_` prefix is redundant because of the module it's in

Reply:
Oh my bad, I figured `state.rs` only made sense while it was the only `_state.rs` file for this actor. I'll change it back.

Reply:
ah, that's a good point, opinionated change then, feel free to leave if you want (just seems asymmetric to me)